src/auspex/pulse_calibration.py

```diff
@@ -342,8 +342,7 @@ def phase_estimation( data_in, vardata_in, verbose=False):
         curGuess = possibles[0]
         if verbose == True:
             print('Current Guess: %f'%(curGuess))
```

Comment:
I have the feeling that there is already a lot of information in `logger.info` that may obscure the most important messages. Maybe we can have a different logging level between `INFO` and `WARNING`?